src/components/List/List.jsx

```diff
@@ -5,20 +5,45 @@ import { useFirebase } from '../../hooks/useFirebase';
 const List = () => {
   const token = localStorage.getItem('token');
 
-  const { getAll } = useFirebase();
+  const { getAll, update } = useFirebase();
 
   const firebasePath = getAll().doc(token).collection('items');
 
   const [value, loading, error] = useCollection(firebasePath);
 
+  const handleCheck = (id, lastDate) => {
+    if (isPassed24Hours(lastDate) === false) {
+      const date = new Date();
+      update(token, id, { lastDate: date });
+    }
+  };
+
+  const isPassed24Hours = (date) => {
+    if (date) {
+      const currentDate = new Date();
+      const itemDate = new Date(date.toDate());
+      itemDate.setDate(itemDate.getDate() + 1);
```

Comment:
Well that was a lot simpler of a solution than I was expecting. Nice job!

Reply:
It took me a while to understand why I had not realized that they were using the methods of the date-fns library. Nice work, I think the timer is very nice.👍🏻